types/wordpress__components/dropdown-menu/index.d.ts

```diff
@@ -33,6 +35,37 @@ declare namespace DropdownMenu {
          * @defaultValue "top center"
          */
         position?: Popover.Position | undefined;
+
+        /**
+         * Object to pass as props to the underlying NavigableMenu component.
+         */
+        menuProps?: Partial<NavigableMenu.Props> | undefined;
+
+        /**
+         * Object to pass as props to the underlying Popover component.
+         */
+        popoverProps?: Partial<Popover.Props> | undefined;
+
+        /**
+         * Object to pass as props to the underlying toggle Button component.
+         */
+        toggleProps?: Partial<Button.ButtonProps> | undefined;
+
+        /**
+         * Whether to disable the arrow down key to open the dropdown menu.
+         * @defaultValue false
+         */
+        disableOpenOnArrowDown?: boolean | undefined;
```

Comment:
Looks like this will be more accurate [than the README](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/dropdown-menu#disableopenonarrowdown) which claims it's an `Object` defaulting to `false` (I suspect boolean is correct).

Reply:
Yeah, that readme value seems to be a copy/paste issue 😄